internal/tracer/exporter.go

```diff
@@ -0,0 +1,146 @@
+package tracer
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	exporttrace "go.opentelemetry.io/otel/sdk/export/trace"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+)
+
+// Exporter does 3 things:
```

Comment:
feel free to tell me to F off, but have you considered using BatchSpanProcessor?
https://github.com/open-telemetry/opentelemetry-go/blob/1ab645fedb49f300145c671bdab3fe57564bf45a/sdk/trace/batch_span_processor.go

most of the batching this class does seems pretty generalizable, and not Tilt-specific